src/index.html

```diff
@@ -6,6 +6,18 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
```

Comment:
Consider adding the `crossorigin` attribute to the `preconnect` link for `fonts.gstatic.com` to improve performance by allowing the browser to make the connection earlier.